src/modules/governance/utils/formatProposal.ts

```diff
@@ -24,7 +24,10 @@ export function formatProposal(proposal: Omit<Proposal, 'values'>) {
 
   const diff = new BigNumber(proposal.forVotes).minus(proposal.againstVotes);
   const voteSum = new BigNumber(proposal.forVotes).plus(proposal.againstVotes);
-  const requiredDiff = voteSum.multipliedBy(new BigNumber(proposal.minimumDiff).dividedBy(100));
+
+  const requiredDiff = new BigNumber(proposal.totalVotingSupply)
+    .multipliedBy(proposal.minimumDiff)
+    .dividedBy(10000);
```

Comment:
should we label the 10000 as a variable so its more clear or leave a comment on how we do the calculation?

Reply:
Yeah, it would be good to move that to a variable and add this [link](https://github.com/aave/governance-v2/blob/master/contracts/governance/ProposalValidator.sol#L191) as a comment so we remember the calculation

Reply:
Feel free to propose a comment. I will accept it. Imo it doesn't make to much sense - perhaps having a comment above the function with a link to the smart contract but no explaining a single random value.

Reply:
https://github.com/aave/interface/pull/907/files#diff-31d08d0f7bfd01ee2d2d0001e717dd0bbc87035fa4774e4eca2b189dcf1f61b1R6